Products/CMFPlone/tests/testCatalogTool.py

```diff
@@ -1083,11 +1083,11 @@ def test_provided(self):
         self.assertTrue(IIndexableObjectWrapper.providedBy(w))
         self.assertTrue(IContentish.providedBy(w))
 
-    def test_getIcon(self):
-        doc = self.doc
-        iconname = doc.getIcon(relative_to_portal=1)
-        wrapped = IndexableObjectWrapper(doc, self.portal.portal_catalog)
-        self.assertEqual(wrapped.getIcon, iconname)
+#     def test_getIcon(self):
```

Comment:
please remove this code; there's no reason to comment it if is not going to be used.